fenix/app/src/main/java/org/mozilla/fenix/shopping/middleware/ReviewQualityCheckVendorsService.kt

```diff
@@ -54,6 +59,8 @@ class DefaultReviewQualityCheckVendorsService(
         }
     }
 
+    override fun getProductVendor(): ProductVendor = productVendors().first()
```

Comment:
Seems like we're defaulting to `AMAZON` for the previews. Should we use `productVendors().first()` there instead or change this to `AMAZON`?

Reply:
since it's just a preview, it doesn't matter a ton since it's just controlling which string appears